man/sd_notify.xml

```diff
@@ -202,6 +202,14 @@
         watchdog is enabled. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>EXTEND_TIMEOUT=1</term>
+
+        <listitem><para>Tells the service manager to set the startup or shutdown service timeout be the watchdog time
+        interval in the future. The service RuntimeMaxSec is also extended if set.
```

Comment:
"The maximum time the service is allowed to run (set by RuntimeMaxSec=) is also extended." (The way this was phrased implied that RuntimeMaxSec itself is extended, which I hope is not true.)

Reply:
also, please always enclose settings in `<varname></varname>`, and suffix them with `=`. i.e. `<varname>RuntimeMaxSec=</varname>`

Reply:
@keszybz - "RuntimeMaxSec itself is extended, which I hope is not true" - this is currently the case. @poettering asked for this in the https://github.com/systemd/systemd/pull/7214#issuecomment-343451927 above. Why isn't this desirable?

Reply:
I wasn't clear. What I meant is that the configuration setting itself should not be changed. And indeed, it's not. Only the timestamp in memory is changed, and the unit itself of its representation on disk is not changed. (We have various code which will update the unit itself, for example when properties are set over dbus, new configuration snippets are written to the file system. This is not the case here).

Reply:
Ok. makes sense. Reworded a lot and changed the implication that settings are changed, only behaviour.